src/utils/Network.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+
+class Network {
+  static getUrl(route) {
+    if (route.indexOf('http://') === 0 || route.indexOf('https://') === 0 || route.indexOf('www.') === 0) {
+      return route;
+    }
+    return `${process.env.REACT_APP_API_HOST}${route}`;
+  }
+
+  static async basicHeaders() {
+    const headers = {};
+
+    // ONLY USE THIS IF YOU WORK IN A BROWSER
+    const token = await window.localStorage.getItem('ACCESS_TOKEN');
+    if (token) {
+      headers.authorization = `Bearer ${token}`;
+    }
+    headers['Content-Type'] = 'application/json';
+    headers['Accept'] = 'application/json';
+
+    return headers;
+  }
+
+  static async errorHandler(error) {
+    if (error.response) {
+      throw {
+        errors : error.response.data.errors,
```

Comment:
Gaat er altijd een .data available zijn? If not kunnen we hier beter een soort van lodash achtige manier op gooien denk ik. _.get(error.response, 'data.errors');

Reply:
Das wel de bedoeling... error.response.data komt van axios zelf, en normaal zijn iCapps error responses altijd gewrapped in een errors object, dus zou die er altijd moeten zijn.. Kzal er een extra check op steken nog, lodash laat ik eruit, dat vind ik nogal overkill voor een boilerplate en enkel deze get

Reply:
Goh ... True ... Ik wil nog wel een case maken voor lodash toe te voegen :D. Los daarvan, denk ik niet dat dit specifiek geeent moet zijn op iCapps backends daar we ook vaak met andere backends werken. Niet?

Reply:
Klopt.. Hmm, maar ik moet er toch iets kunnen insteken? Ik ben er idd wel vanuit gegaan dat we hem connecteren op een iCapps backend, zelfs als het met een lodash get gebeurt, dan is het nog iCapps specifiek?

Reply:
jah we moeten wel ergens vanuit gaan he. Maar ik zou gewoon nog eentje dieper de check doen en even afchecken of de 'data' defined is.

Reply:
In dat geval: Done ;-)